### DISCUSSION TITLE
Vehicle mode does not stop after bluetooth is disconnected, causing 'speak readings' to continue.

### DISCUSSION THREAD

**Discussion by Gferrar123** [NightscoutFoundation/xDrip#3217] [Bug reports]
Vehicle mode does not stop after bluetooth is disconnected, causing 'speak readings' to continue.
Running Version 2023.11.14 
Samsung Galaxy 23 with Android 14
Dexcom G6
I have recreated it with two different bluetooth pairings and the issue has been confirmed by @Navid200 
Any fix would be great, but I would love to see a vehicle mode trigger based on Android Auto running, rather than a specific bluetooth connection.

**Comment by Navid200** [comment]
Thanks for reporting this.
Let's wait a few days for others to see this.  
If this is really an issue, I will ask you to convert it to an issue.

Thanks

**Comment by Navid200** [comment]
@Gferrar123 
Would you please click on "Create issue from discussion" in the right pane?
Thanks

**Comment by richard-dennehy** [comment]
I also have this issue since upgrading to Android 14

**Comment by Gferrar123** [comment]
@navid200  Done!  Is there anything else I need to do?  It's my first time on github.

**Reply by Navid200** [reply]
No, nothing else for now.

Thanks

Android 14 has caused a few issues.  They will be addressed.

**Reply by StevenUK1974** [reply]
Hi,

Same issue here after upgrading to an S24 running Android 14.

Is there an ETA for a fix please? Bit embarrassing at the pub quiz last night when my phone carried on speaking readings despite being in silent mode!! 😬😂

Thanks
Steven

**Reply by Navid200** [reply]
There is an open issues related to this:
https://github.com/NightscoutFoundation/xDrip/issues/3219

Any of the developers could be working on this.
Everyone is welcome to work on any of the open issues.
I can only provide estimates for what I personally am working on may be completed and I am not working on this.

**Comment by vanelsberg** [comment]
Was seeing the same (Samsung Galaxy S22 Ultra) which was already running Android 14. I did get an OS update recently that might have introduced the problem...

Will try xDrip release of May 12, 2024 as sugested at #3219
